src/views/Dashboard/Customer/Home/RequestTest/AgreementDialog.vue

```diff
@@ -1,70 +1,75 @@
-    v-card
+  v-dialog.staking-dialog(:value="show" width="400" persistent rounded )
+    v-card.staking-dialog__dialog
       v-app-bar(flat dense color="white" )
-        v-toolbar-title(class="title mt-8" v-if="agreement") Staking Coin Agreement
-        v-toolbar-title(class="title mt-8" v-if="!agreement") Staking Coin Agreement
+        v-toolbar-title.staking-dialog__title Staking Coin
         v-spacer
-        v-btn( class="mt-8" icon @click="closeDialog")
+        v-btn(icon @click="closeDialog")
           v-icon mdi-close
-      v-card(class="ms-4 me-4" style="background-color: #F5F7F9; font-family: Raleway ")
-        v-card-text(class="mb-4 mt-10")
-          div(style="font-size: 12px" class="me-3" )
-            div 1. Your funds will be staked in smart contract.
-            div 2. You can unstake your funds at any moment. Your funds will be withdrawn to your wallet in 6 days or 144 hours from the moment you confirm an unstake request. 
-            div 3. You will receive DBIO as a reward upon a successful request test when the service is ready. DBIO reward percentage will be determined by DAOGenics.
-
-      v-card-text(class="mt-4 pb-0 text-subtitle-1")
-        div(class="text-body-1 mt-10")
-          div.mb-3 Enter your Amount
-            v-row
-              v-col(
-                cols="12"
-                sm="9"
-              )
-                v-text-field(
-                  v-model="amount"
-                  outlined
-                )
-              
-              v-col(
-                cols="12"
-                sm="3"
-              )
-                v-text-field(
-                  value="DBIO"
-                  outlined
-                  disabled 
-                )
-
-        v-checkbox(class="mt-5" v-model="agree")
-          template(v-slot:label)
-            b I have read and agree to the <a> terms and conditions </a>
-
-      v-card-actions(class="px-6 pb-4")
-        v-btn(
+
+      .staking-dialog__card
+        .staking-dialog__card-text 
+          .staking-dialog__card-text-content 1. There's no locking period. Your fund can be unstaked anytime with your consent but there will be a period of 6 days to process it.
+          .staking-dialog__card-text-content 2. Upon receiving your test result, you will be rewarded with DBIO as a token of gratitude for using our service. By unstaking your fund, you will lose this privilege.
+          .staking-dialog__card-text-content 3. You will receive notification when requested service is available. You can proceed to request a test. If the staked amount is bigger than the service price, you will get refund for overpayment. If staked amount is smaller than the service price, to complete the purchase, you should pay for the outstanding amount to complete the purchase.
+
+      .staking-dialog__input
+        .staking-dialog__input-label Enter your Amount 
+        ui-debio-input.staking-dialog__input-field(
+          :rules="amountRules"
```

Comment:
just curious, are rules work here?

Reply:
yes it works! :D
![image](https://user-images.githubusercontent.com/87380683/146141064-43fbae0c-7f8c-4751-a65c-f5623435aebf.png)

Reply:
yes it works! :D
![image](https://user-images.githubusercontent.com/87380683/146141064-43fbae0c-7f8c-4751-a65c-f5623435aebf.png)

Reply:
Okay, feel free to merge @ciniiia 👍🏼